9. Partitions, LVM/HW9.1 Malasherifov Vitalii.txt

```diff
@@ -0,0 +1,171 @@
+	n
+	34
+	+2G
+	8300
+1.2. Create a 512MB partition on /dev/sdc of type "Linux swap"
+	n
+	default
+	+512M
+	8200
+	
+	After all - write partitions
+	w
+1.3. Format the 2GB partition with an XFS file system
+	[root@vm01 vit]# lsblk 
+	NAME                 MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+	sda                    8:0    0    8G  0 disk 
+	├─sda1                 8:1    0    1G  0 part /boot
+	└─sda2                 8:2    0    7G  0 part 
+	  ├─centos_vm01-root 253:0    0  6.2G  0 lvm  /
+	  └─centos_vm01-swap 253:1    0  820M  0 lvm  [SWAP]
+	sdb                    8:16   0    8G  0 disk 
+	└─sdb1                 8:17   0  300M  0 part /mnt/part1
+	sdc                    8:32   0    5G  0 disk 
+	├─sdc1                 8:33   0    2G  0 part #to XFS
+	└─sdc2                 8:34   0  512M  0 part #for SWAP
+	sr0                   11:0    1 1024M  0 rom  
+	[root@vm01 vit]# mkfs -t xfs /dev/sdc1
+	meta-data=/dev/sdc1              isize=512    agcount=4, agsize=131072 blks
+			 =                       sectsz=512   attr=2, projid32bit=1
+			 =                       crc=1        finobt=0, sparse=0
+	data     =                       bsize=4096   blocks=524288, imaxpct=25
+			 =                       sunit=0      swidth=0 blks
+	naming   =version 2              bsize=4096   ascii-ci=0 ftype=1
+	log      =internal log           bsize=4096   blocks=2560, version=2
+			 =                       sectsz=512   sunit=0 blks, lazy-count=1
+	realtime =none                   extsz=4096   blocks=0, rtextents=0
+1.4. Initialize 512MB partition as swap space
+	root@vm01 vit]# mkswap /dev/sdc2 
+	Setting up swapspace version 1, size = 524284 KiB
+	no label, UUID=15123931-5d1f-4c73-a84a-b4f8fa15df47
+	Add to /etc/fstab
+	/dev/sdc2       swap    swap    defaults        0 0
+	[root@vm01 vit]# swapon -a
+	[root@vm01 vit]# swapon -s
+	Filename                                Type            Size    Used    Priority
+	/dev/dm-1                               partition       839676  0       -2
+	/dev/sdc2                               partition       524284  0       -3
+1.5. Configure the newly created XFS file system to persistently mount at /backup
+	Add to /etc/fstab
+	/dev/sdc1 /mount        xfs     defaults        0 0
+	
+	mkdir /backup
+	mount -a
+	[root@vm01 vit]# lsblk
+	NAME                 MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+	sda                    8:0    0    8G  0 disk 
+	├─sda1                 8:1    0    1G  0 part /boot
+	└─sda2                 8:2    0    7G  0 part 
+	  ├─centos_vm01-root 253:0    0  6.2G  0 lvm  /
+	  └─centos_vm01-swap 253:1    0  820M  0 lvm  [SWAP]
+	sdb                    8:16   0    8G  0 disk 
+	└─sdb1                 8:17   0  300M  0 part /mnt/part1
+	sdc                    8:32   0    5G  0 disk 
+	├─sdc1                 8:33   0    2G  0 part /backup
+	└─sdc2                 8:34   0  512M  0 part [SWAP]
+	sr0                   11:0    1 1024M  0 rom 
+1.6. Configure the newly created swap space to be enabled at boot
```

Comment:
There is no new entry in fstab for new swap partition

Reply:
fixed